code/handlers/trackservers.q

```diff
@@ -333,13 +333,23 @@ reqprocsnotconn:{[requiredprocs]
     not all requiredprocs in exec proctype from .servers.SERVERS where .dotz.liveh[w]
   }
 
+startupdependent:{ 
```

Comment:
Surely this wont work?  The argument declaration has been dropped, so unless [requiredprocs;timeintv] are globals (which they definitely shouldn't be) then this will fail?